server/build/webpack.js

```diff
@@ -187,7 +193,7 @@ export default async function createCompiler (dir, { dev = false, quiet = false
     resolve: {
       modules: [
         nextNodeModulesDir,
-        'node_modules'
+        appNodeModulesDir
```

Comment:
Had to set this, otherwise webpack raise issues that, it can't find `next/*` apis.